### DISCUSSION TITLE
Home page feed customization ("Following", "For you (Beta)", ...) should be synced across browsers and devices

### DISCUSSION THREAD

**Discussion by resolritter** [community/community#52955] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

**User story**

1 - I selected the "For you (Beta)" feed for my home page.
2 - I reloaded the page. I noticed that the "For you (Beta)" feed was still selected even after the reload; i.e. it had become my default.
3 - I switched to a different web browser and logged into GitHub there.
4 - I noticed that my Home page feed customization (step 1) was not available in the new browser.

**Suggestion**

Home page feed customization should be synced across browsers and devices. I don't want to have to redo my selection and filters for each different browser/device that I use.

**Comment by hamidrezamd1380** [comment]
https://realit.ir/
